getting-started.md

```diff
@@ -27,8 +27,7 @@ Let's start with the basics.
      response.end('Hello World!\n');
   });
 
-  // Try retrieving a port from an environment variable, otherwise fallback to 8080.
-  var port = process.env.PORT || 8080;
+  var port = 3000;
```

Comment:
@romuloctba You are right, 8080 seems more sensible here. But if the user has tomcat or apache installed on their machines, and if they are active, then the port 8080 might not be available. That is why I left it to 3000. This can be 8080 as well. I don't have a strong preference anyway.